lib/registry.sh

```
@@ -1,8 +1,10 @@
+# check if alias is registered
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)